vnext/Microsoft.ReactNative/ReactHost/ReactInstanceWin.h

```diff
@@ -186,6 +190,9 @@ class ReactInstanceWin final : public Mso::ActiveObject<IReactInstanceInternal>
 
   std::shared_ptr<facebook::jsi::RuntimeHolderLazyInit> m_jsiRuntimeHolder;
   winrt::Microsoft::ReactNative::JsiRuntime m_jsiRuntime{nullptr};
+
+  static std::mutex s_mutex; // protects access to s_entries
+  static std::vector<ReactInstanceWin *> s_entries;
```

Comment:
>s_entries [](http://example.com/codeflow?start=41&length=9)

nit: it would be nice to have a more descriptive name. E.g. s_instanceRegistry or something else that says that these are all running instances.